spring-boot/admin/src/main/java/eu/coatrack/admin/service/CustomProxyFileGeneratorService.java

```diff
@@ -57,10 +55,6 @@ public class CustomProxyFileGeneratorService {
 
     private static final Logger log = LoggerFactory.getLogger(CustomProxyFileGeneratorService.class);
 
-    // ****************  Config Properties  ****************  
-    @Value("${ygg.proxy.executable-jar.template.url}")
```

Comment:
Why remove this configurable property? It was introduced to allow flexible configuration.